examples/combined_ODNP.py

```diff
@@ -371,13 +372,12 @@
         with h5py.File(
             os.path.normpath(os.path.join(target_directory, filename))
             ) as fp:
-            if nodename in fp.keys():
+            while nodename in fp.keys():
```

Comment:
it keeps going until it finds `temp_##` that doesn't already exist

Reply:
I like this